Dockerfile.amd64

```diff
@@ -24,9 +24,7 @@ LABEL name="Calico CLI tool" \
       maintainer="maintainers@projectcalico.org"
 
 ADD bin/calicoctl-linux-amd64 /calicoctl
-
-RUN mkdir /licenses
```

Comment:
I'm not sure what this was originally for so I've no idea if removing it is sensible.  Do you have any more details on this?

Reply:
I don't know why that file is required. I'll bring back the original file.